src/deploymentinfo.cpp

```diff
@@ -52,3 +54,49 @@ std::optional<Consensus::BuriedDeployment> GetBuriedDeployment(const std::string
     }
     return std::nullopt;
 }
+
+#define FLAG_NAME(flag) {std::string(#flag), SCRIPT_VERIFY_##flag}
+const std::map<std::string, uint32_t> g_verify_flag_names{
+    FLAG_NAME(P2SH),
```

Comment:
If we're exposing this, probably better to call it "BIP16" since p2wsh and p2tr(sometimes) are also p2sh.

Reply:
"Pay to script hash" is the title of BIP16, and we use both "pay-to-script-hash" and "P2SH" as descriptions of the `sh()` descriptor in doc/descriptors.md, so I don't think this is confusing or needs any changes.